### PR TITLE
Ch/cime4rexports

### PR DESCRIPTION
**Pull request by ginihumer**
just some code that enables us to export data in the format of [CIME4R](https://reaction-optimization.caleydoapp.org/) (a tool for visualizing reaction optimization workflows)

### CODE REVIEW

File: scripts/submit-branin.sh [hunk 1]
```diff
@@ -8,4 +8,4 @@
 module load anaconda/3 # replace: load anaconda module
 conda activate al_new  # replace: conda env name
 cd /home/mila/c/christina.humer/activelearning # replace: location of the code
-python main.py --config-name test_branin
\ No newline at end of file
+python main.py --config-name test_branin user=chumer
```
Comment (alexhernandezgarcia) [inline scripts/submit-branin.sh]: You can use `user=$USER` to make it general and valid for any user (provided there is a config file with the user name).

**Review by alexhernandezgarcia — APPROVED** [review]
I can't really provide any good feedback about the details of the plotter part, but on the surface it looks good. Feel free to merge.

File: activelearning/dataset/ocp.py [hunk 2]
```diff
@@ -41,31 +43,41 @@ def get_item_from_index(self, index):
...
     def __getitem__(self, key):
         if isinstance(key, int):
-            x, y = self.get_item_from_index(key)
+            x, y, idcs_in_dataset = self.get_item_from_index(key)
         elif isinstance(key, slice):
             start, stop, step = key.indices(len(self))
             x = torch.Tensor([])
             y = torch.Tensor([])
+            idcs_in_dataset = torch.Tensor([])
             for i in range(start, stop, step):
```
Comment (vict0rsch) [inline activelearning/dataset/ocp.py:60]: I would not `concat` sequentially in a `for` loop, but either:

* store in a list and `concat` afterwards
* pre-allocate a large tensor and fill it (~`large[a:b, ...] = somehting`)

File: scripts/submit-branin.sh (hunk 1 above)
Comment (ginihumer) [reply scripts/submit-branin.sh]: good point

File: activelearning/dataset/ocp.py (hunk 2 above)
Comment (ginihumer) [reply activelearning/dataset/ocp.py:60]: does this have an effect on performance? or just bad practice?

**Pull request merged**